elastic-job-lite-console/pom.xml

```diff
@@ -86,6 +86,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
+                <version>2.2-beta-5</version>
```

Comment:
Can we change a stable version instead of the beta version?

Reply:
I have tested all the versions，change maven-assembly-plugin version to 3.3.0.
but it should be id in assembly.xml .please check.